tests/contest/contest/src/tests/time_ns/mod.rs

```diff
@@ -0,0 +1,142 @@
+use anyhow::{Context, Ok, Result};
+use oci_spec::runtime::{
+    LinuxBuilder, LinuxIdMappingBuilder, LinuxNamespace, LinuxNamespaceType, ProcessBuilder, Spec,
+    SpecBuilder,
+};
+use test_framework::{Test, TestGroup, TestResult, test_result};
+
+mod util;
+use crate::tests::time_ns::util::create_time_offset;
+use crate::utils::test_inside_container;
+use crate::utils::test_utils::CreateOptions;
+
+fn create_spec_for_set_times() -> Result<Spec> {
+    let mut default_namespaces: Vec<LinuxNamespace> = oci_spec::runtime::get_default_namespaces();
+    default_namespaces.push(
+        LinuxNamespace::default()
+            .set_typ(LinuxNamespaceType::Time)
+            .to_owned(),
+    );
+
+    let time_offsets = create_time_offset(1337, 3141519, 7881, 2718281);
```

Comment:
At a glance, I find it a bit hard to understand what these arguments represent.
Would it be possible to introduce a small struct (or a similar type) to give them clearer meaning?